### PR TITLE
Better crouch dodge

### PR DESCRIPTION
**Pull request by eps0003**
## Status

**IN DEVELOPMENT**

## Description

As requested, I've reimplemented crouch dodging by using tags to make the code more future-proof. Currently, crouch dodging can be done by pressing down while visibly crouching or while shielding as knight. This can easily be changed by adding the "crouch dodge" tag in the animation states you want.

## Steps to Test or Reproduce

Test 1: Throw a boulder in the air and crouch to dodge it while in different animation states
Test 2: Place a boulder on the ground and crouch grapple through it

### CODE REVIEW

File: Entities/Characters/Scripts/RunnerDefault.as [hunk 1]
```diff
@@ -26,6 +26,7 @@ void onInit(CBlob@ this)
 
 void onTick(CBlob@ this)
 {
+	this.Untag("crouch dodge");
```
Comment (Vikhenzo) [inline Entities/Characters/Scripts/RunnerDefault.as]: Are you untagging and retagging every frame? That doesn't seem great.

Comment (eps0003) [reply Entities/Characters/Scripts/RunnerDefault.as]: What should I be doing instead? Should I untag in every animation state that doesn’t allow crouch dodging?

Comment (FuzzCat927) [reply Entities/Characters/Scripts/RunnerDefault.as]: probably need to tag / untag only when an animation state that doesn't allow starts / ends

Comment (eps0003) [reply Entities/Characters/Scripts/RunnerDefault.as]: That’ll require lots of repetitive lines of code...

**Pull request closed without merging**